apps/demo/emails/google-play-policy-update.tsx

```diff
@@ -98,33 +100,33 @@ export default function Email() {
               <Text style={paragraph}>Connect with us</Text>
               <table>
                 <tr>
```

Comment:
Can you also replace the `table` and `tr`?

Reply:
Yes sure, but just wanna know I need to replace with which tags?